docs/visual-testing/faq.md

```diff
@@ -0,0 +1,39 @@
+---
+id: faq
+title: Sauce Visual Testing FAQ
+sidebar_label: FAQ
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+### Is Sauce Visual's image comparison pixel-based or DOM-based?
```

Comment:
```suggestion
## Is Sauce Visual's image comparison pixel-based or DOM-based?
```

Reply:
We set it to h3 because otherwise the question would become to big